### PR TITLE
Map [Page|ElementHandle|Frame|Locator].Click promises and fix tests

### LINKED ISSUE #707: Map [Page|ElementHandle|Frame|Locator].Click promises and fix tests
This issue is about mapping `Page.Click`, `Frame.Click`, etc. methods in the mapping layer.

See issue #683 for more details.

### PR DESCRIPTION
**Pull request by inancgumus**
Closes #707.

### CODE REVIEW

File: browser/mapping.go [hunk 1]
```diff
@@ -149,7 +149,12 @@ func mapElementHandle(ctx context.Context, vu k6modules.VU, eh api.ElementHandle
 		"objectID":       eh.ObjectID,
 		"boundingBox":    eh.BoundingBox,
 		"check":          eh.Check,
-		"click":          eh.Click,
+		"click": func(opts goja.Value) *goja.Promise {
+			return k6ext.Promise(ctx, func() (any, error) {
+				err := eh.Click(opts)
+				return nil, fmt.Errorf("%w", err)
```
Comment (inancgumus) [inline browser/mapping.go:155]: Forgot: `//nolint:wrapcheck` here. Harmless, but, you know.

File: tests/frame_test.go [hunk 2]
```diff
@@ -57,29 +54,11 @@ func TestFrameDismissDialogBox(t *testing.T) {
...
-
-					return nil
-				}).then(func() {
-					result := p.TextContent("#textField", nil)
-					assert.EqualValues(t, tt+" dismissed", result)
-				})
-
-				return nil
-			})
+			err = p.Click("#clickHere", nil)
```
Comment (inancgumus) [inline tests/frame_test.go:57]: @ankur22 This looks wrong to me, please double check 🤦

File: browser/mapping.go (hunk 1 above)
Comment (ankur22) [reply browser/mapping.go:155]: Maybe make it consistent. I prefer `fmt.Errorf("%w", err)` over `//nolint:wrapcheck`. I leave it you though 👍

**Review by ankur22 — APPROVED** [review]
LGTM

File: browser/mapping.go (hunk 1 above)
Comment (inancgumus) [reply browser/mapping.go:155]: We decided against that in sync back in July and disabled the wrapcheck linter. I'm confused about what changed your mind in that time frame 😄 I prefer the second, the first one is unnecessarily verbose and `%w` should not be preferred unless you want to unwrap the error. I'll go with the wrapcheck.

Comment (ankur22) [reply browser/mapping.go]: I can't remember that far back :) Happy with the disabling of the wrapcheck linter. Why do we need the nolint still if the wrapcheck is disabled?

**Pull request merged**